build/AzurePipelinesTemplates/WindowsAppSDK-PublishInsightsHeader-Steps.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  WindowsAppSDKInsightsSourceDirectory: $(Build.SourcesDirectory)\dev\WindowsAppSDK_Insights
+  WindowsAppSDKInsightsBinariesDirectory: $(Build.SourcesDirectory)\BuildOutput
+
+steps:
+- task: NuGetToolInstaller@1
+  displayName: 'Use NuGet 5.6'
+  inputs:
+    versionSpec: 5.6
+  continueOnError: true
+
+# START of copying files into their final package location.
+
+- task: CopyFiles@2
+  displayName: 'copy WindowsAppSDKInsights.h into output folder'
+  inputs:
+    SourceFolder: '${{ parameters.WindowsAppSDKInsightsSourceDirectory }}'
+    Contents: |
+      WindowsAppSDKInsights.h
+    TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\include'
+    flattenFolders: true
+
+- task: PublishBuildArtifacts@1
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)\fullnuget'
+    ArtifactName: 'FullNuget'
```

Comment:
this is probably handled by others. i tend to think we don't need a separate yml file.

Reply:
Is the comment to merge this into another yml? If so, I want to keep this separate so that we are able to find the steps when we need to.